nativelink-store/src/shard_store.rs

```diff
@@ -167,7 +167,7 @@ impl StoreDriver for ShardStore {
             .collect();
         // Bucket each key into the store that it belongs to.
         keys.iter()
-        .cloned()
+            .cloned()
```

Comment:
I'm not blaming you because it was here for almost two years. It think we could be more efficient than `.cloned().enumerate()`. The only reason I would even bring it up is the fact that `shard_store` really only matters for large workloads.